.github/workflows/run-host-tests.yml

```diff
@@ -56,7 +73,12 @@ jobs:
             apt-get update && apt-get install -y python3-pip rsync
             python -m pip install gcovr
             cd $GITHUB_WORKSPACE/${{inputs.component_path}}
-            gcov `find . -name "*gcda"`
+            component_name=$(basename ${{ inputs.component_path }})
+            if [[ ${{ inputs.idf_version }} == "release-v4.3" ]]; then
+                gcov-8 `find . -name "$component_name*gcda" -printf '%h\n' | head -n 1`/*
+            else
+                gcov `find . -name "$component_name*gcda"`
+            fi
```

Comment:
can we create a simple function in bash, e.g. `get_gdca()` and put it into the `env.sh`?

then we can simply call `gcov get_gdca()` with no ifdef here

Reply:
Okay, so could we find a better way of choosing which method we use for finding the files, other than based on `idf_version`? Maybe `gcov --version`?
Sorry, I just don't find this condition descriptive enough, maybe after adding some comments it might be...?